PopPUNK/assign.py

```diff
@@ -38,10 +38,12 @@ def assign_query(dbFuncs,
                  previous_clustering,
                  external_clustering,
                  core_only,
-                 accessory_only):
-    """Code for assign query mode. Written as a separate function so it can
-    be called by web APIs
-    """
+                 accessory_only,
+                 web = False,
+                 sketch = None):
```

Comment:
I think to keep it all consistent, don't give these arguments a default here, and instead explicitly set them to False and None from the call at L407 of assign.py

Reply:
Also, can you give `sketch` a more informative name? Perhaps `json_sketch`